runtime/commits.go

```diff
@@ -19,14 +19,15 @@ func walkCommits(n *parser.NodeProgram, visitor *RuntimeVisitor) (*TableData, er
 
 	s := n.Child.(*parser.NodeSelect)
 	where := s.Where
+	distinct := s.Distinct
 
 	counter := 1
 	fields := s.Fields
 	if s.WildCard {
 		fields = builder.possibleTables[s.Tables[0]]
 	}
 	resultFields := fields // These are the fields in output with wildcards expanded
-	rows := make([]tableRow, s.Limit)
+	rows := make([]tableRow, 1)
```

Comment:
Did you change the limit on purpose?

Reply:
Changed the initial rows on purpose, limit remains the same. Otherwise we would get repeated values when using the `distinct`. Makes sense?

Is there another way to achieve this?

Reply:
Did you revert it?